app/server.js

```diff
@@ -0,0 +1,38 @@
+
+import express from 'express';
+import fetch from 'isomorphic-fetch';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+import { DaprClient, HttpMethod } from "@dapr/dapr";
+
+const app = express();
+app.use(express.json());
+
+const port = process.env.APP_PORT ?? '3000';
```

Comment:
I know it's a convention but in our use case we want the port to match dapr command, so setting it up to the value used in README should be enough

Reply:
Oki, will update.